grass7/vector/v.in.pygbif/v.in.pygbif.py

```diff
@@ -832,5 +830,12 @@ def main():
 
 # Run the module
 if __name__ == "__main__":
+    try:
+        from osgeo import ogr, osr
+        from osgeo import __version__ as gdal_version
+    except ImportError:
+        grass.fatal(_("Unable to load GDAL Python bindings (requires "
+                      "package 'python-gdal' being installed)"))
```

Comment:
```suggestion
                      "package 'python-gdal' or python library GDAL "
                      "to be installed)."))
```

Reply:
Fixed in the [56c0275](https://github.com/OSGeo/grass-addons/commit/56c02750f30616cc3d5d94a37110e2f37491ec3d).